resources/repos.json

```diff
@@ -1078,5 +1078,23 @@
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/master/#73\">dev #73</a>"
             }
         }
+    },
+    "Apeiros/AlchimiaVitae:master": {
```

Comment:
Your username is `Apeiros-46B`, not `Apeiros`...

Reply:
Sorry, forgot about that